contrasts_kwdyz11.qmd

```diff
@@ -1,24 +1,32 @@
 ---
 title: Contrast Coding of Visual Attention Effects
 engine: julia
-julia:
+julia: 
   exeflags: ["--project", "--threads=auto"]
+
+fig-format: png
 ---
 
+
 ```{julia}
 #| code-fold: true
+using CairoMakie
 using Chain
 using DataFrames
 using MixedModels
+using RCall
 using SMLP2024: dataset
 using StatsBase
 using StatsModels
 
-const progress=false
+CairoMakie.activate!(; type="png")
+
+import ProgressMeter
 ```
 
 # A word of caution {#sec-caution}
-
+Many researchers have pointed out that contrasts should be “tested instead of, rather than as a supplement to, the ordinary ‘omnibus’ F test” (Hays, 1973, p. 601).
```

Comment:
```suggestion
Many researchers have pointed out that contrasts should be "tested instead of, rather than as a supplement to, the ordinary 'omnibus' F test" (Hays, 1973, p. 601).
```